package.json

```diff
@@ -60,11 +60,33 @@
     "lint-fix": "eslint --fix --ext .js,.vue parliament viewer notifiers wiseService",
     "lint-fix-viewer": "eslint --fix --ext .js,.vue viewer",
     "lint-fix-parliament": "eslint --fix --ext .js,.vue parliament",
-    "lint-fix-wise": "eslint --fix --ext .js,.vue wiseService"
+    "lint-fix-wise": "eslint --fix --ext .js,.vue wiseService",
+    "viewer:addtestuser": "cd viewer && node addUser.js -c ../tests/config.test.ini -n testuser admin admin admin --admin --packetSearch",
+    "viewer:build": "cd viewer && node vueapp/build/build.js",
+    "viewer:bundle": "webpack -d --progress --config viewer/vueapp/build/webpack.dev.conf.js",
+    "viewer:start": "cd viewer && node viewer.js -c ../config.ini",
```

Comment:
Does a --prefix flag work here? Then you dont have to run cd everytime

Reply:
I don't think prefix works with the node command. Or maybe I'm not understanding what you're saying

Reply:
Huh, guess it doesnt